install/installer/Dockerfile

```diff
@@ -6,7 +6,9 @@ RUN apk add --no-cache \
         openssh \
         # For PyInquirer
         gcc \
-        musl-dev
+        musl-dev \
```

Comment:
What is that? what purpose?

Reply:
https://en.wikipedia.org/wiki/Musl

[PyInquirer](https://github.com/CITGuru/PyInquirer#question-types) - is cool CLI Q/A interface.  

[`musl-dev`](https://pkgs.alpinelinux.org/package/v3.3/main/x86/musl-dev)`*` Needed for `PyInquirer` dependencies correct install. Without it, get next:

```shell
...
Building wheels for collected packages: PyInquirer, regex
  Building wheel for PyInquirer (setup.py): started
  Building wheel for PyInquirer (setup.py): finished with status 'done'
  Created wheel for PyInquirer: filename=PyInquirer-1.0.3-py3-none-any.whl size=32849 sha256=24a68799f1d249b4f5e52395b1dfca9051440526d94fca40f79fc9faee1c555e
  Stored in directory: /home/cluster.dev/.cache/pip/wheels/bd/f3/88/f02d7b8ae8d125c2ab25d91e2c65fea21a17fb9324371f906a
  Building wheel for regex (setup.py): started
  Building wheel for regex (setup.py): finished with status 'error'
  ERROR: Command errored out with exit status 1:
   command: /usr/local/bin/python -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-85ppdfep/regex/setup.py'"'"'; __file__='"'"'/tmp/pip-install-85ppdfep/regex/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' bdist_wheel -d /tmp/pip-wheel-s0szy82k
       cwd: /tmp/pip-install-85ppdfep/regex/
  Complete output (21 lines):
  running bdist_wheel
  running build
  running build_py
  creating build
  creating build/lib.linux-x86_64-3.8
  creating build/lib.linux-x86_64-3.8/regex
  copying regex_3/__init__.py -> build/lib.linux-x86_64-3.8/regex
  copying regex_3/regex.py -> build/lib.linux-x86_64-3.8/regex
  copying regex_3/_regex_core.py -> build/lib.linux-x86_64-3.8/regex
  copying regex_3/test_regex.py -> build/lib.linux-x86_64-3.8/regex
  running build_ext
  building 'regex._regex' extension
  creating build/temp.linux-x86_64-3.8
  creating build/temp.linux-x86_64-3.8/regex_3
  gcc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -DTHREAD_STACK_SIZE=0x100000 -fPIC -I/usr/local/include/python3.8 -c regex_3/_regex.c -o build/temp.linux-x86_64-3.8/regex_3/_regex.o
  In file included from regex_3/_regex.c:48:
  /usr/local/include/python3.8/Python.h:11:10: fatal error: limits.h: No such file or directory
     11 | #include <limits.h>
        |          ^~~~~~~~~~
  compilation terminated.
  error: command 'gcc' failed with exit status 1
  ----------------------------------------
  ERROR: Failed building wheel for regex
  Running setup.py clean for regex
Successfully built PyInquirer
Failed to build regex
Installing collected packages: smmap, gitdb, gitpython, six, wcwidth, prompt-toolkit, Pygments, regex, PyInquirer
  WARNING: The script pygmentize is installed in '/home/cluster.dev/.local/bin' which is not on PATH.
  Consider adding this directory to PATH or, if you prefer to suppress this warning, use --no-warn-script-location.
    Running setup.py install for regex: started
    Running setup.py install for regex: finished with status 'error'
    ERROR: Command errored out with exit status 1:
     command: /usr/local/bin/python -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-85ppdfep/regex/setup.py'"'"'; __file__='"'"'/tmp/pip-install-85ppdfep/regex/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record /tmp/pip-record-w3i17pmi/install-record.txt --single-version-externally-managed --user --prefix= --compile --install-headers /home/cluster.dev/.local/include/python3.8/regex
         cwd: /tmp/pip-install-85ppdfep/regex/
    Complete output (21 lines):
    running install
    running build
    running build_py
    creating build
    creating build/lib.linux-x86_64-3.8
    creating build/lib.linux-x86_64-3.8/regex
    copying regex_3/__init__.py -> build/lib.linux-x86_64-3.8/regex
    copying regex_3/regex.py -> build/lib.linux-x86_64-3.8/regex
    copying regex_3/_regex_core.py -> build/lib.linux-x86_64-3.8/regex
    copying regex_3/test_regex.py -> build/lib.linux-x86_64-3.8/regex
    running build_ext
    building 'regex._regex' extension
    creating build/temp.linux-x86_64-3.8
    creating build/temp.linux-x86_64-3.8/regex_3
    gcc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -DTHREAD_STACK_SIZE=0x100000 -fPIC -I/usr/local/include/python3.8 -c regex_3/_regex.c -o build/temp.linux-x86_64-3.8/regex_3/_regex.o
    In file included from regex_3/_regex.c:48:
    /usr/local/include/python3.8/Python.h:11:10: fatal error: limits.h: No such file or directory
       11 | #include <limits.h>
          |          ^~~~~~~~~~
    compilation terminated.
    error: command 'gcc' failed with exit status 1
    ----------------------------------------
ERROR: Command errored out with exit status 1: /usr/local/bin/python -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-85ppdfep/regex/setup.py'"'"'; __file__='"'"'/tmp/pip-install-85ppdfep/regex/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record /tmp/pip-record-w3i17pmi/install-record.txt --single-version-externally-managed --user --prefix= --compile --install-headers /home/cluster.dev/.local/include/python3.8/regex Check the logs for full command output.
```

`*` - musl is a C standard library intended for operating systems based on the Linux kernel, released under the MIT License. Source: [Wikipedia](https://en.wikipedia.org/wiki/Musl)